src/test/java/com/google/alpollo/AnalysisTest.java

```diff
@@ -34,14 +43,49 @@
   private static final String LYRICS_SHORT = "I'm the mountain\n"
       + "Rising high\n" + "It's the way that I survived\n" + "I'm the mountain\n" + "Tell my tale\n"
       + "The greatest story's now for sale\n";
+  
+  private ServletContext mockServletContext = mock(ServletContext.class);
+  private HttpServletRequest request = mock(HttpServletRequest.class);
+  private HttpServletResponse response = mock(HttpServletResponse.class);
+  private SentimentServlet sentimentServletUnderTest;
+  private EntityServlet entityServletUnderTest;
+  private StringWriter responseWriter;
+
   private static final String LYRICS_WITH_METADATA = "Google has found me some nice results!";
   private static final String EMPTY_STRING = "";
   private static final String WIKI_LINK_GOOGLE = "https://en.wikipedia.org/wiki/Google";
 
 
+  @Before
+  public void setUp() throws Exception {
+    sentimentServletUnderTest = new SentimentServlet() {
+      @Override
+      public ServletContext getServletContext() {
+        return mockServletContext;
+      }
+    };
+
+    entityServletUnderTest = new EntityServlet() {
+      @Override
+      public ServletContext getServletContext() {
+        return mockServletContext;
+      }
+    };
+
+    when(mockServletContext.getResourceAsStream(anyString())).thenAnswer(
+        (Answer<InputStream>) invocation -> new FileInputStream(CONFIG_FILE_PATH));
```

Comment:
Here you could be a bit less generic, and use the argument of getResourceAsStream in the answer, as what you want to change is not the entire thing, but just the root path. That way, if you ever need the ServletContext to access other resources, the mockServletContext will still work